var/spack/repos/builtin/packages/gegelati/package.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Gegelati(CMakePackage):
+    """Class to add the a TPG library(gegelati) into Spack"""
+
+    homepage = "https://github.com/gegelati/gegelati"
+    url = "https://github.com/gegelati/gegelati/archive/refs/tags/v1.2.0.tar.gz"
+
+    # notify when the package is updated.
```

Comment:
This comment is no longer necessary.  Suggest removing it *unless* you really want to keep it.

Reply:
No worries. I want to keep it.
Thank you :)